### PR TITLE
feat: add OpenFeature Web Provider docs

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

**Comment by jonathannorris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #488** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/devcycle-docs/488?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/devcycle-docs/488?utm_source=stack-comment).

File: docs/sdk/client-side-sdks/javascript/javascript-openfeature.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+sidebar_position: 5
+description: How to implement the OpenFeature Web Provider
+sidebar_custom_props: { icon: toggle-off }
+---
+
+# OpenFeature Web Provider
+
+[OpenFeature](https://openfeature.dev/) is an open standard that provides a vendor-agnostic, community-driven API for feature flagging that works with DevCycle.
+
+DevCycle provides a Javascript implementation of the OpenFeature Web Provider interface, if you prefer to use the OpenFeature APIs to interface with DevCycle.
```
Comment (JamieSinn) [inline docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:13]: might want to add a reference to the nodejs one in case you get confused here?

Comment (jonathannorris) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:13]: I think it is pretty clear that it's for web no?

Comment (JamieSinn) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:13]: I may be the dumb one - but I equate JS = nodejs all the time when looking for docs

**Review by JamieSinn — APPROVED** [review]

File: docs/sdk/client-side-sdks/javascript/javascript-openfeature.md (hunk 1 above)
Comment (jonathannorris) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:13]: the title is `Web Provider`, I think that's pretty explicit

**Pull request merged**